eZ/Publish/Core/IO/TolerantIOService.php

```diff
@@ -105,6 +113,20 @@ public function loadBinaryFileByUri($binaryFileUri)
     }
 
     /**
+     * @param $binaryFileUri
+     *
+     * @return MissingBinaryFile
+     */
+    private function createMissingBinaryFileByUri($binaryFileUri)
```

Comment:
@bdunogier Ok, sure. Then I take it we don't need `createMissingBinaryFile()` either.